### PR TITLE
Plotly Dash - Dash Bio Examples

### PR DESCRIPTION
**Pull request by aditipate**
This PR:

Adds examples from the Dash Bio suite - a suite of bioinformatics components that make it simpler to analyze and visualize bioinformatics data and interact with them in a Dash application.
There are 16 total examples that utilize data/components in categories ranging from molecular, genomic, and chromosome data.
Examples were tested locally and in the Trainee cluster.

### CODE REVIEW

File: tutorials/dash/dash-bio-suite/06-alignmentchart.py
Comment (ktaletsk) [inline tutorials/dash/dash-bio-suite/06-alignmentchart.py:1]: Output looks like raw HTML (not rendered). Do you see the same when you start this dashboard?
image

File: tutorials/dash/dash-bio-suite/16-manhattanplot.py
Comment (ktaletsk) [inline tutorials/dash/dash-bio-suite/16-manhattanplot.py:1]: I am confused. What is the difference between this example and Manhattan plot you have previously created at `tutorials/dash/dash-manhattan-plot/app.py`?

**Review by ktaletsk — CHANGES_REQUESTED** [review]
Everyhting mostly looks good. I was able to launch all of the examples in Notebooks Hub.

Main request here is to download the data from server, check in the data into this folder with Dash examples and load it locally

See my comments for the full list of requests

File: tutorials/dash/dash-bio-suite/01-molecule2dviewer.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import dash_bio as dashbio
+from flask import Flask
+
+server = Flask(__name__)
+app = Dash(server=server)
+
+
+model_data = urlreq.urlopen(
+    'https://git.io/mol2d_buckminsterfullerene.json'
+).read().decode('utf-8')
```
[the remark this answers is not in the document]
Comment (aditipate) [reply tutorials/dash/dash-bio-suite/01-molecule2dviewer.py]: That's a good suggestion. I added all of the data being used in the dash-bio-suite examples and added them to tutorials/dash/dash-bio-suite/data.

File: tutorials/dash/dash-bio-suite/06-alignmentchart.py
Comment (aditipate) [reply tutorials/dash/dash-bio-suite/06-alignmentchart.py:1]: Yes, I'm also getting the same output when I run the dashboard. Just re-ran locally.

File: tutorials/dash/dash-bio-suite/16-manhattanplot.py
Comment (aditipate) [reply tutorials/dash/dash-bio-suite/16-manhattanplot.py:1]: Both examples include the dash manhattan plot + slider and utilize the same data, however my example has an additional component which is the gene summary table. Significant SNP data is passed to the the gene summary table, along with additional info about the significant genes that's been queried from NCBI (e.g summaries).

Comment (ktaletsk) [reply tutorials/dash/dash-bio-suite/16-manhattanplot.py:1]: Ok, if it's the same plot type with the same data, let's replace this example with your example

File: tutorials/dash/dash-bio-suite/06-alignmentchart.py
Comment (ktaletsk) [reply tutorials/dash/dash-bio-suite/06-alignmentchart.py:1]: Could you come up with a fix, so it does not show the raw HTML, but rendered instead?

**Review by ktaletsk — APPROVED** [review]

**Pull request merged**